vllm/config.py

```diff
@@ -50,6 +50,8 @@
     "Qwen2ForCausalLM",
     "Qwen2MoeForCausalLM",
     "QWenLMHeadModel",
+    "InternVLChatModel",
+    "InternLM2ForCausalLM"
 ]
```

Comment:
Let's keep this in alphabetical order.

Reply:
Done

Reply:
I mean that the entire list should remain in alphabetical order.

Reply:
Done